modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/matchers/CompletableFutureMatcher.java

```diff
@@ -174,6 +174,26 @@ public static CompletableFutureMatcher<Object> willFailIn(int time, TimeUnit tim
         return new CompletableFutureMatcher<>(anything(), time, timeUnit, cause);
     }
 
+    /**
+     * Creates a matcher that matches a future that <strong>not</strong> completes decently fast.
+     *
+     * @return matcher.
+     */
+    public static CompletableFutureMatcher<Object> willTimeoutFast() {
+        return willTimeoutIn(1, TimeUnit.SECONDS);
```

Comment:
1 second is "fast"?

Reply:
I can do a quarter/half of a second, for example, for the win os, less than 200 ms may not be enough.

Reply:
set 250 ms